fox_scraper/fox_scraper.py

```diff
@@ -38,6 +40,28 @@ def print_item(self):
         print "item name:", self.item_name
         print "item price: %.2f " %self.item_price
 
+    def print_assert(self):
+        return self.item_img_id + self.item_main_category + self.item_type + self.item_name + str(self.item_price)
+
+
+class FoxRabbitmq(object):
+    def __init__(self):
+        self.connection = pika.BlockingConnection(pika.ConnectionParameters(host='localhost'))
+        self.channel = self.connection.channel()
+        self.channel.exchange_declare(exchange='fox_scrap', type='direct')
+
+#    def establish_rabbitmq_connection(self):
```

Comment:
will do
(when the tabbitmq module is ready - it still in "basic developing" :)  )